bridger-cli/src/utils/epochHandler.test.ts

```diff
@@ -0,0 +1,93 @@
+import { setEpochRange, getBlockNumberFromEpoch, checkForNewEpoch } from "./epochHandler";
+
+describe("epochHandler", () => {
+  describe("setEpochRange", () => {
+    let veaOutbox: any;
+    let startEpoch: number;
+    beforeEach(() => {
+      veaOutbox = {
+        epochNow: jest.fn().mockResolvedValue(10),
+      };
+      startEpoch = 10;
+    });
+    it("should return an array of epoch", async () => {
+      const result = await setEpochRange(veaOutbox, startEpoch);
+      expect(result).toBeDefined();
+      expect(result).toEqual([10]);
+    });
+
+    it("should throw an error if start<current epoch", async () => {
+      startEpoch = 12;
+      await expect(setEpochRange(veaOutbox, startEpoch)).rejects.toThrow("Current epoch is less than start epoch");
```

Comment:
This will be more robust with custom errors, as it also allows you to define the error message only once instead of having it duplicated (in the error and the test)